docs/content_management/forms/create_form_attribute.md

```diff
@@ -118,4 +118,16 @@ In the main menu, click **Content** -> **Forms** -> **Create content**, and sele
 
 You should be able to see the new section in the list of available fields:
 
-![New form](new_form.png)
+![New form attribute](checkbox_with_richtext_description-item.png)
+
+When the attribute settings are edited, the "Description" field is a richtext textarea.
```

Comment:
When editing settings, the "Description" attribute has the Rich Text input.

Reply:
Done in f8fea01e3584f34c2d0b467d3b343af929c308f5